modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -5,17 +5,29 @@
 :clamav-url: http://www.clamav.net/index.html
 :icap-url: https://tools.ietf.org/html/rfc3507
 :c-icap-url: https://sourceforge.net/p/c-icap/wiki/configcicap/
-:kasperski-scanengine-url: https://support.kaspersky.com/ScanEngine/1.0/en-US/199729.htm
+:kasperski-scanengine-url: https://support.kaspersky.com/ScanEngine/1.0/en-US/179682.htm
+:kaspersky-icap-url: https://support.kaspersky.com/ScanEngine/1.0/en-US/179708.htm
+:mcaffee-demo-url: https://www.skyhighsecurity.com/en-us/products/secure-web-gateway.html
+:fortinetsandbox-url: https://www.fortinet.com/de/products/sandbox/fortisandbox
+
+:description: When sharing files, security is a key aspect. The ownCloud Anti-Virus extension helps by protecting against malicious software like trojans or viruses. 
 
 == Introduction
 
-When sharing files, security is a key aspect. The ownCloud {oc-marketplace-url}/apps/files_antivirus[Anti-Virus] extension helps by protecting against malicious software like trojans or viruses. It forwards files that are being uploaded to the ownCloud server to a  malware scanning engine before they are written to the storage. When a file is recognized as malicious, it can be logged and prevented from being uploaded to the server to ensure that files in ownCloud are free of malware. More sophisticated rules may be specified as admin in the ownCloud Webinterface menu:Admin[Settings > Security].
+{description}
+
+You can get the {oc-marketplace-url}/apps/files_antivirus[Anti-Virus] extension from the marketplace. It helps by protecting against malicious software like trojans or viruses. It forwards files that are being uploaded to the ownCloud server to a  malware scanning engine before they are written to the storage. When a file is recognized as malicious, it can be logged and prevented from being uploaded to the server to ensure that files in ownCloud are free of malware. More sophisticated rules may be specified as admin in the ownCloud Webinterface menu:Admin[Settings > Security].
```

Comment:
```suggestion
You can get the {oc-marketplace-url}/apps/files_antivirus[Anti-Virus] extension from the marketplace. It forwards files that are being uploaded to the ownCloud server to a  malware scanning engine before they are written to the storage. When a file is recognized as malicious, it can be logged and prevented from being uploaded to the server to ensure that files in ownCloud are free of malware. More sophisticated rules may be specified as admin in the ownCloud Web interface menu:Admin[Settings > Security].
```

Reply:
Deleted sentence has been moved to the description.